stylesheets/_utility.utilities.scss

```diff
@@ -330,6 +335,12 @@ div.no-results {
     width: auto !important;
 }
 
+%u-margin-top,
+.u-margin-top {
```

Comment:
Any reason for the `em` use here, other utility classes use `px`

Reply:
replaced with $margin-base